packages/gatsby-image/src/index.js

```diff
@@ -295,13 +286,23 @@ class Image extends React.Component {
                 top: 0,
                 bottom: 0,
                 opacity: !this.state.imgLoaded ? 1 : 0,
-                transitionDelay: `0.35s`,
+                transitionDelay: initialDelay,
                 right: 0,
                 left: 0,
               }}
             />
           )}
 
+          {/* Show the blurry base64 image. */}
+          {image.base64 && (
```

Comment:
what's the reason for moving it down?

Reply:
As the PR and commits should explain, it was to fix the layer ordering with `backgroundColor` placeholder, so that it can be used as a fallback with an image placeholder. Presently the image would be covered by the `backgroundColor` placeholder.